### ISSUE TITLE
Environment Variables

### ISSUE THREAD

**Issue by themeteorchef** [themeteorchef/base#85]
The only one coming to mind for this is the `MAIL_URL` at the moment, but it would be good to update the server startup with a loader for this like:

```javascript
let startup = () => {
  _setBrowserPolicies();
  _generateAccounts();
  _setEnvironmentVariables();
};

let _setBrowserPolicies = () => {};

let _generateAccounts = () => Modules.server.generateAccounts();

let _setEnvironmentVariables = () => {
  process.env.MAIL_URL = Meteor.settings.private.MAIL_URL;
};

Modules.server.startup = startup;
```

**Comment by josephdburdick** [comment]
@themeteorchef I actually did something like this recently: https://github.com/josephdburdick/vessel/commit/318bed1a39670e4867659ba6faf68a8e178710c6

**Comment by themeteorchef** [comment]
@josephdburdick neat! Want to do a PR for Base? Only ask is that the pattern above is followed to support multiple environment variables.

**Comment by josephdburdick** [comment]
@themeteorchef yeah that would be really cool. Given that in this context we're only talking about email and there could be many API token/keys/etc, should we leave blanks in the settings file like I have for Mailgun?

**Comment by themeteorchef** [comment]
I would say just for `MAIL_URL` right now, like this:

```
{
  "public": {},
  "private": {
    "MAIL_URL": "" // leave empty
  }
}
```

**Comment by jasonbahl** [comment]
I'm getting the following error when I try and start the app: 

`TypeError: Cannot read property 'MAIL_URL' of undefined`

All I've done is clone the app and try and start it.

**Comment by jasonbahl** [comment]
ignore my last comment. missed starting the app with `--settings settings-development.json`

(https://themeteorchef.com/base/settings/)

Thanks!

**Comment by josephdburdick** [comment]
Issue #91 has been filed with a suggested fix @jasonbahl 
Btw, a faster way to start the app without all that typing is by using `$ npm start` instead.

**Comment by themeteorchef** [comment]
This will be patched and released as `v3.2.1`. Sorry for the confusion @jasonbahl!
